include/braincloud/BrainCloudIdentity.h

```diff
@@ -4,11 +4,13 @@
 
 #include <string>
 
+
 namespace BrainCloud
 {
 	class IServerCallback;
 	class BrainCloudClient;
 	class AuthenticationType;
+    struct AuthenticationIds;
```

Comment:
ignore spacing formatting issue. After this is merged, I'll do another PR to change all tabs to spaces across the entire codebase